lms/templates/problem.html

```diff
@@ -5,38 +5,106 @@
 %>
 
 <%namespace name='static' file='static_content.html'/>
-<h3 class="hd hd-2 problem-header">
+<h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="${ id }-problem-progress" tabindex="-1">
```

Comment:
@cahrens Any idea why we're using `id` here and what if we use `short_id` aka `problem_id` instead of this. Actually i want to refer this heading to the question title and unfortunately we don't have this id at that point, responsetype.py but we do have short_id that is being used to all other places as an identifier.